client/server/epochs.go

```diff
@@ -0,0 +1,48 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/piplabs/story/client/server/utils"
+	"github.com/piplabs/story/client/x/epochs/keeper"
+	epochstypes "github.com/piplabs/story/client/x/epochs/types"
+)
+
+func (s *Server) initEpochsRoute() {
+	s.httpMux.HandleFunc("/epochs/epoch_infos", utils.SimpleWrap(s.aminoCodec, s.GetEpochInfos))
+	s.httpMux.HandleFunc("/epochs/current_epoch/{identifier}", utils.SimpleWrap(s.aminoCodec, s.GetCurrentEpoch))
+}
+
+// GetEpochInfos queries running epochInfos.
+func (s *Server) GetEpochInfos(r *http.Request) (resp any, err error) {
```

Comment:
what are the return of epochInfos? Does it include info on the start or end block of the current epoch?

Reply:
Do you have a sample response of GetEpochInfo?